CONTRIBUTING.md

```diff
@@ -54,39 +54,37 @@ After submitting the pull request, please make sure the Continuous Integration c
 
 To release a specific package:
 
-1. Merge the relevant package release PR
-2. Switch to the default branch `git checkout master`
-3. Pull latest changes `git pull`
-4. Inside the root run `npm install`
-5. Inside the root directory run `npm test` - do not release if linting or tests are failing. This might indicate a bug
-6. Navigate to the specific package directory via `cd packages/packageName`
-7. Run `npm publish`
-
-If a package is released, the other packages that depend on it should increment their version of that package inside
-their own `package.json`, and a new release of them should be made. This means releasing one package usually ends up
-releasing all upstream packages in order.
+1. Merge the relevant package release PR created by `release-please`
+2. Run `npm publish packages/packageName`
+
+Linting and tests will automatically run before publish. If they fail, please stop releasing as this might indicate a
```

Comment:
> If they fail, please stop releasing

Won't the release process halt automatically?

Reply:
Yes it will, good point.
Fixed.